### DISCUSSION TITLE
Github badges

### DISCUSSION THREAD

**Discussion by guizzo** [Il-Libro-Open-Source/book#50] [Idee]
Ciao a tutt* 👋 

Che ne pensate di creare uno o più badges Github riguardanti il progetto?

Potrebbe essere un modo per mettere in risalto il progetto stesso nonché i vari contributori...che dite?

Lascio qui sotto un'esempio fatto in fretta e furia durante lo standup 😅 

<a href='https://github.com/Il-Libro-Open-Source' target="_blank">
  <img alt='Github' src='https://img.shields.io/badge/Open_Source Book Ambassador-100000?style=flat&logo=Github&logoColor=white&labelColor=313131&color=313131'/>
</a>

EDIT: magari potremmo metterci anche, se possibile, il contatore delle stars cliccabile

**Comment by Cadienvan** [comment]
Mi piace moltissimo l'idea! Dobbiamo solo capire il wording da usare, per il resto mi piace molto!

**Comment by corradopetrelli** [comment]
Uuuuu mi piace! Che ne pensate? @Il-Libro-Open-Source/relations-group

**Reply by Cadienvan** [reply]
Personalmente come dicevo l'idea mi piace. @GuidoPenta @JeyDi

**Comment by halfpass25** [comment]
Facciamolo, se ne riparlava stasera su Telegram e vediamo di rendere la cosa "combinata" con LinkedIn, scopo massima visibilità del progetto 👌💪👊

**Comment by corradopetrelli** [comment]
Sennò che ne pensate di questo?

[![](https://img.shields.io/badge/Open%20Source%20Book%20Ambassador-📖-red.svg)](https://github.com/Il-Libro-Open-Source/book)

**Reply by Cadienvan** [reply]
Molto carino! Non so se preferisco il logo Github o l'emoji del libro..

**Reply by guizzo** [reply]
Si molto carino anche questo, mi piacciono entrambi 😄

**Comment by Cadienvan** [comment]
@guizzo @corradopetrelli che ne dite se proponiamo questa cosa nel gruppo Telegram e prendiamo una decisione a riguardo?

**Reply by corradopetrelli** [reply]
Va benissimo!

**Reply by guizzo** [reply]
Si certo, no problema 👍

**Comment by Cadienvan** [comment]
Chiudo in quanto la scelta è stata fatta e riportata nelle comunicazioni telegram.
